syft/event/parsers/parsers.go

```diff
@@ -94,6 +94,24 @@ func ParseFileDigestsCatalogingStarted(e partybus.Event) (progress.StagedProgres
 	return prog, nil
 }
 
+func ParseFileIndexingStarted(e partybus.Event) (string, progress.StagedProgressable, error) {
```

Comment:
We haven't included unit tests for the ETU / eventing code previously --I think this will change in the near future when we revisit some of the ETUI patterns.